FirebaseAuth/Tests/Unit/FIROAuthProviderTests.m

```diff
@@ -91,17 +101,16 @@
     @brief A fake callback URL containing a fake response URL.
  */
 static NSString *const kFakeRedirectURLResponseURL =
-    @"com.googleusercontent.apps.123456://firebase"
-     "auth/"
-     "link?deep_link_id=https%3A%2F%2Fexample.firebaseapp.com%2F__%2Fauth%2Fcallback%3FauthType"
-     "%3DsignInWithRedirect%26link%3D";
+    @"://firebaseauth/"
```

Comment:
the scheme is missing

Reply:
Amended the comment to clarify -- this the URL without the scheme, since the scheme can be either the reverse client ID or the encoded app ID.

Reply:
Sounds good.